src/app/counter/counter.component.html

```diff
@@ -1 +1,8 @@
-<p>counter works!</p>
+<section class="container d-block w-25 p-1  bg-secondary text-white rounded" style="margin-top: 15%">
```

Comment:
Avoid inline-styling. Make classes in the respective CSS file and use those classes.

Reply:
okay